website/src/css/custom.css

```diff
@@ -107,3 +107,65 @@ svg.lightToggleIcon_node_modules-\@docusaurus-theme-classic-lib-next-theme-Color
 li.footer__item a svg {
   display: inline;
 }
+
+/* mo changes 20 oct 2022 */
+
+.alert {
+  max-width: 1500px;
+  margin: 2em auto;
+}
+
+html[data-theme='dark'] .navbar {
+  background-color: #1b1b1d !important;
+}
+
+.navbar,
+.title-font,
+.text-base {
+  font-family: 'Montserrat' !important;
+}
```

Comment:
I think this can go away with https://github.com/containers/podman-desktop/commit/c94ef04b01fa64c597aa3d0bca5491935cb84b3f

the font is the default font for the whole page

Reply:
dropped this!